services/minespace-web/src/components/pages/Incidents/IncidentPage.js

```diff
@@ -146,7 +145,7 @@ export const IncidentPage = (props) => {
 
   useEffect(() => {
     if (incident.status_code) {
-      setIsFinalReviewStage(POST_SUBMISSION_INCIDENT_STATUSES.includes(incident.status_code));
+      setIsFinalReviewStage(incident.status_code !== "DFT");
```

Comment:
Won't this set `IsFinalReviewStage` to be true at the stage before it's been saved as draft?  I know a few things are dependent on that (including the side menu that appears in the final stage)

Reply:
Nope! By default it's just undefined, so it won't pass the `if (incident.status_code)` guard.